components/Hyperlinks.js

```diff
@@ -1,7 +1,69 @@
 import gql from 'graphql-tag';
-import CircularProgress from '@material-ui/core/CircularProgress';
+import { Box, SvgIcon, CircularProgress } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
 import { useQuery } from '@apollo/react-hooks';
 
+const useStyles = makeStyles(theme => ({
+  root: {
+    padding: '12px 16px',
+    margin: '0 8px 8px 0',
+    background: theme.palette.secondary[50],
+    borderRadius: 8,
+    '& h1': {
+      overflow: 'hidden',
+      textOverflow: 'ellipsis',
+      whiteSpace: 'nowrap',
+      margin: 0,
+      marginBottom: 10,
+      maxWidth: 'inherit',
```

Comment:
font-size should be 14. Currently it's slightly larger than 14px due to Google chrome's user agent stylesheet
![image](https://user-images.githubusercontent.com/108608/82119147-bd0beb00-97ae-11ea-955e-a002fa2b86c8.png)